apps/website/src/pages/_global.astro

```diff
@@ -250,5 +260,16 @@ const { content = {} } = Astro.props;
         element.setAttribute('tabindex', '0');
       });
     </script> -->
+    <script>
```

Comment:
would it make sense to move all the algolia stuff (html, css, js) into a separate astro component?

Reply:
Ohh, right, made a separate component called AlgoliaSearch. [`3f5a944` (#1365)](https://github.com/iTwin/iTwinUI/pull/1365/commits/3f5a944a61a28319b1f319510c649c2860e4d67b)